packages/admin/admin/src/dataGrid/DataGridColumnsManagement.sc.ts

```diff
@@ -0,0 +1,49 @@
+import { css, List as MuiList, ListItem as MuiListItem, Switch as MuiSwitch, Typography } from "@mui/material";
+
+import { createComponentSlot } from "../helpers/createComponentSlot";
+import { type DataGridColumnsManagementClassKey } from "./DataGridColumnsManagement";
+
+export const Root = createComponentSlot("div")<DataGridColumnsManagementClassKey>({
+    componentName: "DataGridColumnsManagement",
+    slotName: "root",
+})();
+
+export const List = createComponentSlot(MuiList)<DataGridColumnsManagementClassKey>({
+    componentName: "DataGridColumnsManagement",
+    slotName: "list",
+})();
+
+export const ListItem = createComponentSlot(MuiListItem)<DataGridColumnsManagementClassKey>({
+    componentName: "DataGridColumnsManagement",
+    slotName: "listItem",
+})(css`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+`);
+
+export const Switch = createComponentSlot(MuiSwitch)<DataGridColumnsManagementClassKey>({
+    componentName: "DataGridColumnsManagement",
+    slotName: "switch",
+})();
+
+export const ListItemTitle = createComponentSlot(Typography)<DataGridColumnsManagementClassKey>({
```

Comment:
Shouldn't we use [ListItemText](https://mui.com/material-ui/api/list-item-text/) for this?

Reply:
Then we need to wait for @jamesricky here as well.

Reply:
we 4 sure can use the `ListItemText` here, but I do not see a benefit when using the `ListItemText` over a normal `Typography`.